debian/wott-agent-self-update.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
```

Comment:
Add in 

```
# vim: tabstop=4 shiftwidth=4 softtabstop=4
# -*- sh-basic-offset: 4 -*-

set -euo pipefail
IFS=$'\n\t'
```

Reply:
Why do we need to set `IFS` here and why `\n\t`?

Reply:
It's just something that I've used over the years. I forgot the exact reason for this, but in general it should harden the bash scripts.

Reply:
This is not only useless in our case but also harmful. Let's not do this.

Reply:
Care to elaborate?

Reply:
It affects tokenizing. You don't expect different tokenizing when you're writing a script unless you really need this.
Default value for IFS is `' \t\n'`. Why would you need to change it?

Reply:
Alright, fair, let's leave IFS as is.